AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/bookings-list/bookings-list.component.spec.ts

```diff
@@ -909,18 +887,6 @@ describe('BookingsListComponent', () => {
         });
     });
 
-    it('should onClear (if workAllocation feature on)', () => {
```

Comment:
this test is still relevant, as work allocation is always on

Reply:
reinstated